pandas/core/tools/datetimes.py

```diff
@@ -486,6 +486,10 @@ def _convert_listlike(arg, box, format, name=None, tz=tz):
                     arg=arg,
                     origin=origin))
 
+        if origin.tzinfo is not None:
```

Comment:
this incorrectly raises on strings that are convertible to a Timestamp.

change to something like

```
# the existing try block, but move the subtraction out
try:
    offset = tslib.Timestamp(origin)
except OutOfBounds:
    ....
except ValueError:
  ...
```

then
```
if offset.tz is not None:
    raise ValueError # nice message about timezones

offset -= tslib.Timestamp(0)
```